components/dashboard/src/service/service-mock.ts

```diff
@@ -217,15 +217,15 @@ const gitpodServiceMock = createServiceMock({
                 id: "g1-standard",
                 category: "GENERAL PURPOSE",
                 displayName: "Standard",
-                description: "Up to 4 vCPU, 8GB memory, 30GB disk",
+                description: "Up to 4 cores, 8GB RAM, 30GB storage",
```

Comment:
I know this is just a mock test file—could someone also update the class description on the database? Also, does cores describe accurately enough what we offer? See also discussion in https://github.com/gitpod-io/website/issues/3340. Cc @atduarte @kylos101

Reply:
Re-posting from https://github.com/gitpod-io/website/issues/3340#issuecomment-1411938219:

> Not entirely sure about the cores terms as vCPUs can describe something different, especially since we using containers and not VMs under the hood but seems to describe accurately enough what we offer and sounds less scary or technical. 💭

Reply:
Cores is slightly incorrect, we should change the website to vCPUs.
```suggestion
                description: "Up to 4 vCPUs, 8GB RAM, 30GB storage",
```

Reply:
One additional idea would be to rename storage to SSD—as they are more performance—but not sure I'm convinced 🤔 
```suggestion
                description: "Up to 4 vCPUs, 8GB RAM, 30GB SSD",
```

Reply:
I'd opt for "storage" for now, just because it does not matter that much atm, and we don't create expectations.

Reply:
Yes, please! Let's keep it as storage.

Reply:
**question**: @atduarte Could we also keep the **cores** term as it's less scary to users? I know it can be slightly inaccurate, but keeping this as is would also remove the need to update the references on the website, etc. Also, if we make the change for the `vCPUs` term we need to update this in the database, right? 

I'll leave the final call to you. 🏓

Reply:
Let's do `cores`! Thanks, @Furisto, for your opinion [[1](https://gitpod.slack.com/archives/C02F19UUW6S/p1675423559937379?thread_ts=1675419194.133049&cid=C02F19UUW6S)].

Reply:
![perfection](https://user-images.githubusercontent.com/120486/216608174-53b4244d-0e7e-4360-9634-1618b96c5e0f.gif)